lessons/Early-Math/exponent-expressions-with-negatives/langs/en-us/index.adoc

```diff
@@ -0,0 +1,100 @@
+= Evaluating Exponent Expressions with Negative Values
+
+@description{Students use Circles of Evaluation to analyze the structure of expressions with exponents and negative values.}
+
+@lesson-prereqs{evaluating-exponents}
+
+@keywords{Circle of Evaluation, }
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Determine the order in which we negate and apply exponents in expressions both with and without grouping symbols.
+* Evaluate expressions involving exponents and negative values.
+* Evaluate exponent expressions with negatives and variables.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Circles of Evaluation to analyse the structure of expressions with exponents and negative values.
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+|===
+
+
+== Exponent Expressions with Negatives @duration{30 minutes}
+
+=== Overview
+
+Students apply Circles of Evaluation to analyze expressions with exponents and negatives.
+
+=== Launch
+
+[.lesson-instruction]
+--
+- Look closely at each of these expressions, considering the effect of the negation and the exponent on the outcome: @math{-3^2}, @math{(-3)^2}, and @math{-(3^2)}
```

Comment:
I'd suggest reordering such that (-3)^2 comes first because it is the most unmistakable example and might lead them to more easily recognize whether the others match the behavior.